ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1221,49 +1221,30 @@ def test_frontend_method(
     # Run testing
     class_name = class_name.split(".")
     ins_class = ivy.functional.frontends.__dict__[frontend]
+    frontend_class = importlib.import_module(frontend)
     for c_n in class_name:
         ins_class = getattr(ins_class, c_n)
+        frontend_class = getattr(frontend_class, c_n)
     ins = ins_class(*args_constructor, **kwargs_constructor)
     ret, ret_np_flat = get_ret_and_flattened_np_array(
         ins.__getattribute__(method_name), *args_method, **kwargs_method
     )
 
     # Compute the return with a Ground Truth backend
-    ivy.set_backend(ground_truth_backend)
-    args_gt_constructor, kwargs_gt_constructor, _, _, _ = create_args_kwargs(
-        args_np=args_np_constructor,
-        arg_np_vals=con_arg_np_vals,
-        args_idxs=con_args_idxs,
-        kwargs_np=kwargs_np_constructor,
-        kwarg_np_vals=con_kwarg_np_vals,
-        kwargs_idxs=con_kwargs_idxs,
-        input_dtypes=input_dtypes_init,
-        as_variable_flags=as_variable_flags_init,
-        native_array_flags=native_array_flags_init,
-    )
-    args_gt_method, kwargs_gt_method, _, _, _ = create_args_kwargs(
-        args_np=args_np_method,
-        arg_np_vals=met_arg_np_vals,
-        args_idxs=met_args_idxs,
-        kwargs_np=kwargs_np_method,
-        kwarg_np_vals=met_kwarg_np_vals,
-        kwargs_idxs=met_kwargs_idxs,
-        input_dtypes=input_dtypes_method,
-        as_variable_flags=as_variable_flags_method,
-        native_array_flags=native_array_flags_method,
-    )
-    ins_gt = ins_class(*args_gt_constructor, **kwargs_gt_constructor)
-    ret_from_gt, ret_np_from_gt_flat = get_ret_and_flattened_np_array(
-        ins_gt.__getattribute__(method_name), *args_gt_method, **kwargs_gt_method
-    )
+    ivy.set_backend(frontend)
+    ins_gt = frontend_class(*args_constructor, **kwargs_constructor)
```

Comment:
One small note, otherwise lgtm! we should be converting `args` and `kwargs` to a native arrays, (ike here: https://github.com/unifyai/ivy/blob/4ee63edb384d2a37c87dbba5b46642f4d08f8b85/ivy_tests/test_ivy/helpers/function_testing.py#L600

I suppose NumPy doesn't throw an exception because all frameworks returns a np array because of the method `__array__`, but I'm assuming if we test a class method for a framework like torch, this will break. Can you do a brief testing locally?